[2023-05-23T22:11:43.805+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: algolia_test.load scheduled__2019-04-02T00:00:00+00:00 [queued]>
[2023-05-23T22:11:43.822+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: algolia_test.load scheduled__2019-04-02T00:00:00+00:00 [queued]>
[2023-05-23T22:11:43.823+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-05-23T22:11:43.824+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-05-23T22:11:43.824+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-05-23T22:11:43.839+0000] {taskinstance.py:1383} INFO - Executing <Task(_PythonDecoratedOperator): load> on 2019-04-02 00:00:00+00:00
[2023-05-23T22:11:43.847+0000] {standard_task_runner.py:55} INFO - Started process 386 to run task
[2023-05-23T22:11:43.861+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'algolia_test', 'load', 'scheduled__2019-04-02T00:00:00+00:00', '--job-id', '80', '--raw', '--subdir', 'DAGS_FOLDER/algolia-dag.py', '--cfg-path', '/tmp/tmpxq8adz90']
[2023-05-23T22:11:43.866+0000] {standard_task_runner.py:83} INFO - Job 80: Subtask load
[2023-05-23T22:11:43.890+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:545: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  option = self._get_environment_variables(deprecated_key, deprecated_section, key, section)

[2023-05-23T22:11:43.967+0000] {task_command.py:376} INFO - Running <TaskInstance: algolia_test.load scheduled__2019-04-02T00:00:00+00:00 [running]> on host 7bd2786c5ce5
[2023-05-23T22:11:44.057+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=algolia_test
AIRFLOW_CTX_TASK_ID=load
AIRFLOW_CTX_EXECUTION_DATE=2019-04-02T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-04-02T00:00:00+00:00
[2023-05-23T22:11:44.059+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:204: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2023-05-23T22:11:44.078+0000] {taskinstance.py:1851} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/decorators/base.py", line 188, in execute
    return_value = super().execute(context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 193, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/algolia-dag.py", line 57, in load
    cursor.execute('DROP TABLE algolia_tests')
psycopg2.errors.UndefinedTable: table "algolia_tests" does not exist

[2023-05-23T22:11:44.095+0000] {taskinstance.py:1406} INFO - Marking task as FAILED. dag_id=algolia_test, task_id=load, execution_date=20190402T000000, start_date=20230523T221143, end_date=20230523T221144
[2023-05-23T22:11:44.110+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 80 for task load (table "algolia_tests" does not exist
; 386)
[2023-05-23T22:11:44.148+0000] {local_task_job.py:159} INFO - Task exited with return code 1
[2023-05-23T22:11:44.168+0000] {taskinstance.py:2623} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-05-23T22:13:19.408+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: algolia_test.load scheduled__2019-04-02T00:00:00+00:00 [queued]>
[2023-05-23T22:13:19.423+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: algolia_test.load scheduled__2019-04-02T00:00:00+00:00 [queued]>
[2023-05-23T22:13:19.426+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-05-23T22:13:19.427+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-05-23T22:13:19.429+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-05-23T22:13:19.451+0000] {taskinstance.py:1383} INFO - Executing <Task(_PythonDecoratedOperator): load> on 2019-04-02 00:00:00+00:00
[2023-05-23T22:13:19.461+0000] {standard_task_runner.py:55} INFO - Started process 528 to run task
[2023-05-23T22:13:19.476+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'algolia_test', 'load', 'scheduled__2019-04-02T00:00:00+00:00', '--job-id', '98', '--raw', '--subdir', 'DAGS_FOLDER/algolia-dag.py', '--cfg-path', '/tmp/tmpochc1i0l']
[2023-05-23T22:13:19.482+0000] {standard_task_runner.py:83} INFO - Job 98: Subtask load
[2023-05-23T22:13:19.508+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:545: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  option = self._get_environment_variables(deprecated_key, deprecated_section, key, section)

[2023-05-23T22:13:19.579+0000] {task_command.py:376} INFO - Running <TaskInstance: algolia_test.load scheduled__2019-04-02T00:00:00+00:00 [running]> on host 7bd2786c5ce5
[2023-05-23T22:13:19.653+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=algolia_test
AIRFLOW_CTX_TASK_ID=load
AIRFLOW_CTX_EXECUTION_DATE=2019-04-02T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-04-02T00:00:00+00:00
[2023-05-23T22:13:19.656+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:204: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2023-05-23T22:13:19.689+0000] {taskinstance.py:1851} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/decorators/base.py", line 188, in execute
    return_value = super().execute(context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 193, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/algolia-dag.py", line 55, in load
    df.to_sql('algolia_tests', engine)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/core/generic.py", line 2882, in to_sql
    method=method,
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/io/sql.py", line 728, in to_sql
    **engine_kwargs,
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/io/sql.py", line 1758, in to_sql
    dtype=dtype,
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/io/sql.py", line 1650, in prep_table
    table.create()
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/io/sql.py", line 858, in create
    raise ValueError(f"Table '{self.name}' already exists.")
ValueError: Table 'algolia_tests' already exists.
[2023-05-23T22:13:19.710+0000] {taskinstance.py:1406} INFO - Marking task as FAILED. dag_id=algolia_test, task_id=load, execution_date=20190402T000000, start_date=20230523T221319, end_date=20230523T221319
[2023-05-23T22:13:19.719+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 98 for task load (Table 'algolia_tests' already exists.; 528)
[2023-05-23T22:13:19.762+0000] {local_task_job.py:159} INFO - Task exited with return code 1
[2023-05-23T22:13:19.795+0000] {taskinstance.py:2623} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-05-23T22:28:58.775+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: algolia_test.load scheduled__2019-04-02T00:00:00+00:00 [queued]>
[2023-05-23T22:28:58.783+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: algolia_test.load scheduled__2019-04-02T00:00:00+00:00 [queued]>
[2023-05-23T22:28:58.784+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-05-23T22:28:58.785+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-05-23T22:28:58.786+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-05-23T22:28:58.802+0000] {taskinstance.py:1383} INFO - Executing <Task(_PythonDecoratedOperator): load> on 2019-04-02 00:00:00+00:00
[2023-05-23T22:28:58.816+0000] {standard_task_runner.py:55} INFO - Started process 1412 to run task
[2023-05-23T22:28:58.829+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'algolia_test', 'load', 'scheduled__2019-04-02T00:00:00+00:00', '--job-id', '123', '--raw', '--subdir', 'DAGS_FOLDER/algolia-dag.py', '--cfg-path', '/tmp/tmpdewiapb4']
[2023-05-23T22:28:58.831+0000] {standard_task_runner.py:83} INFO - Job 123: Subtask load
[2023-05-23T22:28:58.850+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:545: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  option = self._get_environment_variables(deprecated_key, deprecated_section, key, section)

[2023-05-23T22:28:58.913+0000] {task_command.py:376} INFO - Running <TaskInstance: algolia_test.load scheduled__2019-04-02T00:00:00+00:00 [running]> on host 7bd2786c5ce5
[2023-05-23T22:28:58.995+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=algolia_test
AIRFLOW_CTX_TASK_ID=load
AIRFLOW_CTX_EXECUTION_DATE=2019-04-02T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-04-02T00:00:00+00:00
[2023-05-23T22:28:58.998+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:204: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2023-05-23T22:28:59.014+0000] {logging_mixin.py:137} INFO - Index(['Unnamed: 0', 'index', 'id', 'shop_domain', 'application_id',
       'autocomplete_enabled', 'user_created_at_least_one_qr',
       'nbr_merchandised_queries', 'nbrs_pinned_items', 'showing_logo',
       'has_changed_sort_orders', 'analytics_enabled', 'use_metafields',
       'nbr_metafields', 'use_default_colors', 'show_products',
       'instant_search_enabled', 'instant_search_enabled_on_collection',
       'only_using_faceting_on_collection', 'use_merchandising_for_collection',
       'index_prefix', 'indexing_paused', 'install_channel', 'export_date',
       'has_specific_prefix'],
      dtype='object')
[2023-05-23T22:28:59.099+0000] {taskinstance.py:1851} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1783, in _execute_context
    cursor, statement, parameters, context
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/dialects/postgresql/psycopg2.py", line 957, in do_executemany
    **kwargs
  File "/home/airflow/.local/lib/python3.7/site-packages/psycopg2/extras.py", line 1299, in execute_values
    cur.execute(b''.join(parts))
psycopg2.errors.UndefinedColumn: column "level_0" of relation "algolia_tests" does not exist
LINE 1: INSERT INTO algolia_tests (level_0, "Unnamed: 0", index, id,...
                                   ^


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/decorators/base.py", line 188, in execute
    return_value = super().execute(context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 193, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/algolia-dag.py", line 60, in load
    df.to_sql('algolia_tests', engine, if_exists='append')
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/core/generic.py", line 2882, in to_sql
    method=method,
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/io/sql.py", line 728, in to_sql
    **engine_kwargs,
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/io/sql.py", line 1770, in to_sql
    **engine_kwargs,
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/io/sql.py", line 1350, in insert_records
    raise err
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/io/sql.py", line 1340, in insert_records
    table.insert(chunksize=chunksize, method=method)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/io/sql.py", line 967, in insert
    exec_insert(conn, keys, chunk_iter)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/io/sql.py", line 882, in _execute_insert
    conn.execute(self.table.insert(), data)
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1289, in execute
    return meth(self, multiparams, params, _EMPTY_EXECUTION_OPTS)
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/sql/elements.py", line 326, in _execute_on_connection
    self, multiparams, params, execution_options
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1491, in _execute_clauseelement
    cache_hit=cache_hit,
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1846, in _execute_context
    e, statement, parameters, cursor, context
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 2027, in _handle_dbapi_exception
    sqlalchemy_exception, with_traceback=exc_info[2], from_=e
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/util/compat.py", line 207, in raise_
    raise exception
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1783, in _execute_context
    cursor, statement, parameters, context
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/dialects/postgresql/psycopg2.py", line 957, in do_executemany
    **kwargs
  File "/home/airflow/.local/lib/python3.7/site-packages/psycopg2/extras.py", line 1299, in execute_values
    cur.execute(b''.join(parts))
sqlalchemy.exc.ProgrammingError: (psycopg2.errors.UndefinedColumn) column "level_0" of relation "algolia_tests" does not exist
LINE 1: INSERT INTO algolia_tests (level_0, "Unnamed: 0", index, id,...
                                   ^

[SQL: INSERT INTO algolia_tests (level_0, "Unnamed: 0", index, id, shop_domain, application_id, autocomplete_enabled, user_created_at_least_one_qr, nbr_merchandised_queries, nbrs_pinned_items, showing_logo, has_changed_sort_orders, analytics_enabled, use_metafields, nbr_metafields, use_default_colors, show_products, instant_search_enabled, instant_search_enabled_on_collection, only_using_faceting_on_collection, use_merchandising_for_collection, index_prefix, indexing_paused, install_channel, export_date, has_specific_prefix) VALUES (%(level_0)s, %(UnnamedC 0)s, %(index)s, %(id)s, %(shop_domain)s, %(application_id)s, %(autocomplete_enabled)s, %(user_created_at_least_one_qr)s, %(nbr_merchandised_queries)s, %(nbrs_pinned_items)s, %(showing_logo)s, %(has_changed_sort_orders)s, %(analytics_enabled)s, %(use_metafields)s, %(nbr_metafields)s, %(use_default_colors)s, %(show_products)s, %(instant_search_enabled)s, %(instant_search_enabled_on_collection)s, %(only_using_faceting_on_collection)s, %(use_merchandising_for_collection)s, %(index_prefix)s, %(indexing_paused)s, %(install_channel)s, %(export_date)s, %(has_specific_prefix)s)]
[parameters: ({'level_0': 0, 'UnnamedC 0': 0, 'index': 0, 'id': '1d948de7-ea94-4e00-b784-1b598c97dbaa', 'shop_domain': 'himself-hair-southern.myshopify.com', 'application_id': 'HM2DF8P5KS', 'autocomplete_enabled': True, 'user_created_at_least_one_qr': False, 'nbr_merchandised_queries': 0, 'nbrs_pinned_items': '[0]', 'showing_logo': True, 'has_changed_sort_orders': False, 'analytics_enabled': False, 'use_metafields': False, 'nbr_metafields': 9.0, 'use_default_colors': True, 'show_products': False, 'instant_search_enabled': False, 'instant_search_enabled_on_collection': False, 'only_using_faceting_on_collection': False, 'use_merchandising_for_collection': False, 'index_prefix': 'shopify_already_', 'indexing_paused': False, 'install_channel': 'marketplace', 'export_date': '2019-04-02', 'has_specific_prefix': True}, {'level_0': 1, 'UnnamedC 0': 1, 'index': 1, 'id': '9e1209d3-6a7d-45fa-b330-a452316f9dcc', 'shop_domain': 'professional-really-music.myshopify.com', 'application_id': 'WTE6P6DQ90', 'autocomplete_enabled': True, 'user_created_at_least_one_qr': False, 'nbr_merchandised_queries': 0, 'nbrs_pinned_items': '[0]', 'showing_logo': True, 'has_changed_sort_orders': False, 'analytics_enabled': False, 'use_metafields': False, 'nbr_metafields': 11.0, 'use_default_colors': True, 'show_products': False, 'instant_search_enabled': False, 'instant_search_enabled_on_collection': False, 'only_using_faceting_on_collection': False, 'use_merchandising_for_collection': False, 'index_prefix': 'shopify_', 'indexing_paused': False, 'install_channel': 'marketplace', 'export_date': '2019-04-02', 'has_specific_prefix': False}, {'level_0': 2, 'UnnamedC 0': 2, 'index': 2, 'id': 'bceefd16-ebb1-4a26-9080-38cd65b6f5ae', 'shop_domain': 'movie-than-about.myshopify.com', 'application_id': '9EMV0SZBNG', 'autocomplete_enabled': True, 'user_created_at_least_one_qr': False, 'nbr_merchandised_queries': 0, 'nbrs_pinned_items': '[0]', 'showing_logo': True, 'has_changed_sort_orders': False, 'analytics_enabled': False, 'use_metafields': False, 'nbr_metafields': 4.0, 'use_default_colors': True, 'show_products': False, 'instant_search_enabled': False, 'instant_search_enabled_on_collection': False, 'only_using_faceting_on_collection': False, 'use_merchandising_for_collection': False, 'index_prefix': 'shopify_', 'indexing_paused': False, 'install_channel': 'marketplace', 'export_date': '2019-04-02', 'has_specific_prefix': False}, {'level_0': 3, 'UnnamedC 0': 3, 'index': 3, 'id': '67bc3189-e1b1-4628-beb6-4dc288729c20', 'shop_domain': 'growth-this-identify.myshopify.com', 'application_id': '9TCETJH192', 'autocomplete_enabled': True, 'user_created_at_least_one_qr': False, 'nbr_merchandised_queries': 0, 'nbrs_pinned_items': '[0]', 'showing_logo': True, 'has_changed_sort_orders': False, 'analytics_enabled': False, 'use_metafields': False, 'nbr_metafields': 3.0, 'use_default_colors': True, 'show_products': False, 'instant_search_enabled': False, 'instant_search_enabled_on_collection': False, 'only_using_faceting_on_collection': False, 'use_merchandising_for_collection': False, 'index_prefix': 'shopify_his_chance_husband_', 'indexing_paused': False, 'install_channel': 'marketplace', 'export_date': '2019-04-02', 'has_specific_prefix': True}, {'level_0': 4, 'UnnamedC 0': 4, 'index': 4, 'id': 'fc91131e-1175-49a5-be3d-514816f365fd', 'shop_domain': 'do-forget-majority.myshopify.com', 'application_id': '83QW0XS6TQ', 'autocomplete_enabled': True, 'user_created_at_least_one_qr': False, 'nbr_merchandised_queries': 0, 'nbrs_pinned_items': '[0]', 'showing_logo': False, 'has_changed_sort_orders': False, 'analytics_enabled': False, 'use_metafields': False, 'nbr_metafields': 1.0, 'use_default_colors': True, 'show_products': False, 'instant_search_enabled': False, 'instant_search_enabled_on_collection': False, 'only_using_faceting_on_collection': False, 'use_merchandising_for_collection': False, 'index_prefix': 'shopify_behind_', 'indexing_paused': False, 'install_channel': 'marketplace', 'export_date': '2019-04-02', 'has_specific_prefix': True}, {'level_0': 5, 'UnnamedC 0': 5, 'index': 5, 'id': 'a7b0e244-d0b1-4153-ae8d-98e9cd0f272e', 'shop_domain': 'might-beautiful-art.myshopify.com', 'application_id': 'H9AKAWXJ6V', 'autocomplete_enabled': True, 'user_created_at_least_one_qr': False, 'nbr_merchandised_queries': 0, 'nbrs_pinned_items': '[0]', 'showing_logo': True, 'has_changed_sort_orders': False, 'analytics_enabled': False, 'use_metafields': False, 'nbr_metafields': 13.0, 'use_default_colors': True, 'show_products': False, 'instant_search_enabled': False, 'instant_search_enabled_on_collection': False, 'only_using_faceting_on_collection': False, 'use_merchandising_for_collection': False, 'index_prefix': 'shopify_arrive_land_everything_', 'indexing_paused': False, 'install_channel': 'marketplace', 'export_date': '2019-04-02', 'has_specific_prefix': True}, {'level_0': 6, 'UnnamedC 0': 6, 'index': 6, 'id': '03096398-15dc-4807-8fec-889d9697ec51', 'shop_domain': 'industry-age-base.myshopify.com', 'application_id': 'UQO5QXK4AV', 'autocomplete_enabled': True, 'user_created_at_least_one_qr': True, 'nbr_merchandised_queries': 3, 'nbrs_pinned_items': '[1, 2, 2]', 'showing_logo': True, 'has_changed_sort_orders': False, 'analytics_enabled': True, 'use_metafields': True, 'nbr_metafields': None, 'use_default_colors': True, 'show_products': False, 'instant_search_enabled': True, 'instant_search_enabled_on_collection': True, 'only_using_faceting_on_collection': False, 'use_merchandising_for_collection': False, 'index_prefix': 'shopify_', 'indexing_paused': False, 'install_channel': 'marketplace', 'export_date': '2019-04-02', 'has_specific_prefix': False}, {'level_0': 7, 'UnnamedC 0': 7, 'index': 7, 'id': '7dd67bc9-a94a-46c9-9e73-c6e4a9f7af7a', 'shop_domain': 'class-laugh-record.myshopify.com', 'application_id': 'NMFNXL64BX', 'autocomplete_enabled': True, 'user_created_at_least_one_qr': False, 'nbr_merchandised_queries': 0, 'nbrs_pinned_items': '[0]', 'showing_logo': False, 'has_changed_sort_orders': False, 'analytics_enabled': False, 'use_metafields': True, 'nbr_metafields': 18.0, 'use_default_colors': True, 'show_products': False, 'instant_search_enabled': False, 'instant_search_enabled_on_collection': False, 'only_using_faceting_on_collection': False, 'use_merchandising_for_collection': False, 'index_prefix': 'shopify_', 'indexing_paused': False, 'install_channel': 'marketplace', 'export_date': '2019-04-02', 'has_specific_prefix': False}  ... displaying 10 of 1142 total bound parameter sets ...  {'level_0': 1140, 'UnnamedC 0': 1140, 'index': 1206, 'id': '453c5901-25ac-4251-8af5-d833982a1d65', 'shop_domain': 'group-red-range.myshopify.com', 'application_id': 'TMB3QF5RXL', 'autocomplete_enabled': True, 'user_created_at_least_one_qr': False, 'nbr_merchandised_queries': 0, 'nbrs_pinned_items': '[0]', 'showing_logo': True, 'has_changed_sort_orders': True, 'analytics_enabled': False, 'use_metafields': False, 'nbr_metafields': 14.0, 'use_default_colors': True, 'show_products': True, 'instant_search_enabled': True, 'instant_search_enabled_on_collection': False, 'only_using_faceting_on_collection': False, 'use_merchandising_for_collection': False, 'index_prefix': 'shopify_', 'indexing_paused': False, 'install_channel': 'unpublished', 'export_date': '2019-04-02', 'has_specific_prefix': False}, {'level_0': 1141, 'UnnamedC 0': 1141, 'index': 1207, 'id': '577be9b8-7f52-4625-b479-aba19b1de36f', 'shop_domain': 'other-attention-cover.myshopify.com', 'application_id': 'RC89EL9QQD', 'autocomplete_enabled': True, 'user_created_at_least_one_qr': False, 'nbr_merchandised_queries': 0, 'nbrs_pinned_items': '[0]', 'showing_logo': False, 'has_changed_sort_orders': True, 'analytics_enabled': False, 'use_metafields': False, 'nbr_metafields': 4.0, 'use_default_colors': True, 'show_products': False, 'instant_search_enabled': False, 'instant_search_enabled_on_collection': False, 'only_using_faceting_on_collection': True, 'use_merchandising_for_collection': False, 'index_prefix': 'shopify_', 'indexing_paused': False, 'install_channel': 'unpublished', 'export_date': '2019-04-02', 'has_specific_prefix': False})]
(Background on this error at: https://sqlalche.me/e/14/f405)
[2023-05-23T22:28:59.155+0000] {taskinstance.py:1406} INFO - Marking task as FAILED. dag_id=algolia_test, task_id=load, execution_date=20190402T000000, start_date=20230523T222858, end_date=20230523T222859
[2023-05-23T22:28:59.190+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 123 for task load ((psycopg2.errors.UndefinedColumn) column "level_0" of relation "algolia_tests" does not exist
LINE 1: INSERT INTO algolia_tests (level_0, "Unnamed: 0", index, id,...
                                   ^

[SQL: INSERT INTO algolia_tests (level_0, "Unnamed: 0", index, id, shop_domain, application_id, autocomplete_enabled, user_created_at_least_one_qr, nbr_merchandised_queries, nbrs_pinned_items, showing_logo, has_changed_sort_orders, analytics_enabled, use_metafields, nbr_metafields, use_default_colors, show_products, instant_search_enabled, instant_search_enabled_on_collection, only_using_faceting_on_collection, use_merchandising_for_collection, index_prefix, indexing_paused, install_channel, export_date, has_specific_prefix) VALUES (%(level_0)s, %(UnnamedC 0)s, %(index)s, %(id)s, %(shop_domain)s, %(application_id)s, %(autocomplete_enabled)s, %(user_created_at_least_one_qr)s, %(nbr_merchandised_queries)s, %(nbrs_pinned_items)s, %(showing_logo)s, %(has_changed_sort_orders)s, %(analytics_enabled)s, %(use_metafields)s, %(nbr_metafields)s, %(use_default_colors)s, %(show_products)s, %(instant_search_enabled)s, %(instant_search_enabled_on_collection)s, %(only_using_faceting_on_collection)s, %(use_merchandising_for_collection)s, %(index_prefix)s, %(indexing_paused)s, %(install_channel)s, %(export_date)s, %(has_specific_prefix)s)]
[parameters: ({'level_0': 0, 'UnnamedC 0': 0, 'index': 0, 'id': '1d948de7-ea94-4e00-b784-1b598c97dbaa', 'shop_domain': 'himself-hair-southern.myshopify.com', 'application_id': 'HM2DF8P5KS', 'autocomplete_enabled': True, 'user_created_at_least_one_qr': False, 'nbr_merchandised_queries': 0, 'nbrs_pinned_items': '[0]', 'showing_logo': True, 'has_changed_sort_orders': False, 'analytics_enabled': False, 'use_metafields': False, 'nbr_metafields': 9.0, 'use_default_colors': True, 'show_products': False, 'instant_search_enabled': False, 'instant_search_enabled_on_collection': False, 'only_using_faceting_on_collection': False, 'use_merchandising_for_collection': False, 'index_prefix': 'shopify_already_', 'indexing_paused': False, 'install_channel': 'marketplace', 'export_date': '2019-04-02', 'has_specific_prefix': True}, {'level_0': 1, 'UnnamedC 0': 1, 'index': 1, 'id': '9e1209d3-6a7d-45fa-b330-a452316f9dcc', 'shop_domain': 'professional-really-music.myshopify.com', 'application_id': 'WTE6P6DQ90', 'autocomplete_enabled': True, 'user_created_at_least_one_qr': False, 'nbr_merchandised_queries': 0, 'nbrs_pinned_items': '[0]', 'showing_logo': True, 'has_changed_sort_orders': False, 'analytics_enabled': False, 'use_metafields': False, 'nbr_metafields': 11.0, 'use_default_colors': True, 'show_products': False, 'instant_search_enabled': False, 'instant_search_enabled_on_collection': False, 'only_using_faceting_on_collection': False, 'use_merchandising_for_collection': False, 'index_prefix': 'shopify_', 'indexing_paused': False, 'install_channel': 'marketplace', 'export_date': '2019-04-02', 'has_specific_prefix': False}, {'level_0': 2, 'UnnamedC 0': 2, 'index': 2, 'id': 'bceefd16-ebb1-4a26-9080-38cd65b6f5ae', 'shop_domain': 'movie-than-about.myshopify.com', 'application_id': '9EMV0SZBNG', 'autocomplete_enabled': True, 'user_created_at_least_one_qr': False, 'nbr_merchandised_queries': 0, 'nbrs_pinned_items': '[0]', 'showing_logo': True, 'has_changed_sort_orders': False, 'analytics_enabled': False, 'use_metafields': False, 'nbr_metafields': 4.0, 'use_default_colors': True, 'show_products': False, 'instant_search_enabled': False, 'instant_search_enabled_on_collection': False, 'only_using_faceting_on_collection': False, 'use_merchandising_for_collection': False, 'index_prefix': 'shopify_', 'indexing_paused': False, 'install_channel': 'marketplace', 'export_date': '2019-04-02', 'has_specific_prefix': False}, {'level_0': 3, 'UnnamedC 0': 3, 'index': 3, 'id': '67bc3189-e1b1-4628-beb6-4dc288729c20', 'shop_domain': 'growth-this-identify.myshopify.com', 'application_id': '9TCETJH192', 'autocomplete_enabled': True, 'user_created_at_least_one_qr': False, 'nbr_merchandised_queries': 0, 'nbrs_pinned_items': '[0]', 'showing_logo': True, 'has_changed_sort_orders': False, 'analytics_enabled': False, 'use_metafields': False, 'nbr_metafields': 3.0, 'use_default_colors': True, 'show_products': False, 'instant_search_enabled': False, 'instant_search_enabled_on_collection': False, 'only_using_faceting_on_collection': False, 'use_merchandising_for_collection': False, 'index_prefix': 'shopify_his_chance_husband_', 'indexing_paused': False, 'install_channel': 'marketplace', 'export_date': '2019-04-02', 'has_specific_prefix': True}, {'level_0': 4, 'UnnamedC 0': 4, 'index': 4, 'id': 'fc91131e-1175-49a5-be3d-514816f365fd', 'shop_domain': 'do-forget-majority.myshopify.com', 'application_id': '83QW0XS6TQ', 'autocomplete_enabled': True, 'user_created_at_least_one_qr': False, 'nbr_merchandised_queries': 0, 'nbrs_pinned_items': '[0]', 'showing_logo': False, 'has_changed_sort_orders': False, 'analytics_enabled': False, 'use_metafields': False, 'nbr_metafields': 1.0, 'use_default_colors': True, 'show_products': False, 'instant_search_enabled': False, 'instant_search_enabled_on_collection': False, 'only_using_faceting_on_collection': False, 'use_merchandising_for_collection': False, 'index_prefix': 'shopify_behind_', 'indexing_paused': False, 'install_channel': 'marketplace', 'export_date': '2019-04-02', 'has_specific_prefix': True}, {'level_0': 5, 'UnnamedC 0': 5, 'index': 5, 'id': 'a7b0e244-d0b1-4153-ae8d-98e9cd0f272e', 'shop_domain': 'might-beautiful-art.myshopify.com', 'application_id': 'H9AKAWXJ6V', 'autocomplete_enabled': True, 'user_created_at_least_one_qr': False, 'nbr_merchandised_queries': 0, 'nbrs_pinned_items': '[0]', 'showing_logo': True, 'has_changed_sort_orders': False, 'analytics_enabled': False, 'use_metafields': False, 'nbr_metafields': 13.0, 'use_default_colors': True, 'show_products': False, 'instant_search_enabled': False, 'instant_search_enabled_on_collection': False, 'only_using_faceting_on_collection': False, 'use_merchandising_for_collection': False, 'index_prefix': 'shopify_arrive_land_everything_', 'indexing_paused': False, 'install_channel': 'marketplace', 'export_date': '2019-04-02', 'has_specific_prefix': True}, {'level_0': 6, 'UnnamedC 0': 6, 'index': 6, 'id': '03096398-15dc-4807-8fec-889d9697ec51', 'shop_domain': 'industry-age-base.myshopify.com', 'application_id': 'UQO5QXK4AV', 'autocomplete_enabled': True, 'user_created_at_least_one_qr': True, 'nbr_merchandised_queries': 3, 'nbrs_pinned_items': '[1, 2, 2]', 'showing_logo': True, 'has_changed_sort_orders': False, 'analytics_enabled': True, 'use_metafields': True, 'nbr_metafields': None, 'use_default_colors': True, 'show_products': False, 'instant_search_enabled': True, 'instant_search_enabled_on_collection': True, 'only_using_faceting_on_collection': False, 'use_merchandising_for_collection': False, 'index_prefix': 'shopify_', 'indexing_paused': False, 'install_channel': 'marketplace', 'export_date': '2019-04-02', 'has_specific_prefix': False}, {'level_0': 7, 'UnnamedC 0': 7, 'index': 7, 'id': '7dd67bc9-a94a-46c9-9e73-c6e4a9f7af7a', 'shop_domain': 'class-laugh-record.myshopify.com', 'application_id': 'NMFNXL64BX', 'autocomplete_enabled': True, 'user_created_at_least_one_qr': False, 'nbr_merchandised_queries': 0, 'nbrs_pinned_items': '[0]', 'showing_logo': False, 'has_changed_sort_orders': False, 'analytics_enabled': False, 'use_metafields': True, 'nbr_metafields': 18.0, 'use_default_colors': True, 'show_products': False, 'instant_search_enabled': False, 'instant_search_enabled_on_collection': False, 'only_using_faceting_on_collection': False, 'use_merchandising_for_collection': False, 'index_prefix': 'shopify_', 'indexing_paused': False, 'install_channel': 'marketplace', 'export_date': '2019-04-02', 'has_specific_prefix': False}  ... displaying 10 of 1142 total bound parameter sets ...  {'level_0': 1140, 'UnnamedC 0': 1140, 'index': 1206, 'id': '453c5901-25ac-4251-8af5-d833982a1d65', 'shop_domain': 'group-red-range.myshopify.com', 'application_id': 'TMB3QF5RXL', 'autocomplete_enabled': True, 'user_created_at_least_one_qr': False, 'nbr_merchandised_queries': 0, 'nbrs_pinned_items': '[0]', 'showing_logo': True, 'has_changed_sort_orders': True, 'analytics_enabled': False, 'use_metafields': False, 'nbr_metafields': 14.0, 'use_default_colors': True, 'show_products': True, 'instant_search_enabled': True, 'instant_search_enabled_on_collection': False, 'only_using_faceting_on_collection': False, 'use_merchandising_for_collection': False, 'index_prefix': 'shopify_', 'indexing_paused': False, 'install_channel': 'unpublished', 'export_date': '2019-04-02', 'has_specific_prefix': False}, {'level_0': 1141, 'UnnamedC 0': 1141, 'index': 1207, 'id': '577be9b8-7f52-4625-b479-aba19b1de36f', 'shop_domain': 'other-attention-cover.myshopify.com', 'application_id': 'RC89EL9QQD', 'autocomplete_enabled': True, 'user_created_at_least_one_qr': False, 'nbr_merchandised_queries': 0, 'nbrs_pinned_items': '[0]', 'showing_logo': False, 'has_changed_sort_orders': True, 'analytics_enabled': False, 'use_metafields': False, 'nbr_metafields': 4.0, 'use_default_colors': True, 'show_products': False, 'instant_search_enabled': False, 'instant_search_enabled_on_collection': False, 'only_using_faceting_on_collection': True, 'use_merchandising_for_collection': False, 'index_prefix': 'shopify_', 'indexing_paused': False, 'install_channel': 'unpublished', 'export_date': '2019-04-02', 'has_specific_prefix': False})]
(Background on this error at: https://sqlalche.me/e/14/f405); 1412)
[2023-05-23T22:28:59.263+0000] {local_task_job.py:159} INFO - Task exited with return code 1
[2023-05-23T22:28:59.309+0000] {taskinstance.py:2623} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-05-23T22:33:40.002+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: algolia_test.load scheduled__2019-04-02T00:00:00+00:00 [queued]>
[2023-05-23T22:33:40.013+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: algolia_test.load scheduled__2019-04-02T00:00:00+00:00 [queued]>
[2023-05-23T22:33:40.014+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-05-23T22:33:40.014+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-05-23T22:33:40.014+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-05-23T22:33:40.047+0000] {taskinstance.py:1383} INFO - Executing <Task(_PythonDecoratedOperator): load> on 2019-04-02 00:00:00+00:00
[2023-05-23T22:33:40.056+0000] {standard_task_runner.py:55} INFO - Started process 1722 to run task
[2023-05-23T22:33:40.079+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'algolia_test', 'load', 'scheduled__2019-04-02T00:00:00+00:00', '--job-id', '140', '--raw', '--subdir', 'DAGS_FOLDER/algolia-dag.py', '--cfg-path', '/tmp/tmprjtphmxc']
[2023-05-23T22:33:40.088+0000] {standard_task_runner.py:83} INFO - Job 140: Subtask load
[2023-05-23T22:33:40.111+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:545: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  option = self._get_environment_variables(deprecated_key, deprecated_section, key, section)

[2023-05-23T22:33:40.184+0000] {task_command.py:376} INFO - Running <TaskInstance: algolia_test.load scheduled__2019-04-02T00:00:00+00:00 [running]> on host 7bd2786c5ce5
[2023-05-23T22:33:40.287+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=algolia_test
AIRFLOW_CTX_TASK_ID=load
AIRFLOW_CTX_EXECUTION_DATE=2019-04-02T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-04-02T00:00:00+00:00
[2023-05-23T22:33:40.291+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:204: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2023-05-23T22:33:40.312+0000] {logging_mixin.py:137} INFO - Index(['id', 'id.1', 'shop_domain', 'application_id', 'autocomplete_enabled',
       'user_created_at_least_one_qr', 'nbr_merchandised_queries',
       'nbrs_pinned_items', 'showing_logo', 'has_changed_sort_orders',
       'analytics_enabled', 'use_metafields', 'nbr_metafields',
       'use_default_colors', 'show_products', 'instant_search_enabled',
       'instant_search_enabled_on_collection',
       'only_using_faceting_on_collection', 'use_merchandising_for_collection',
       'index_prefix', 'indexing_paused', 'install_channel', 'export_date',
       'has_specific_prefix'],
      dtype='object')
[2023-05-23T22:33:40.487+0000] {taskinstance.py:1851} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1783, in _execute_context
    cursor, statement, parameters, context
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/dialects/postgresql/psycopg2.py", line 957, in do_executemany
    **kwargs
  File "/home/airflow/.local/lib/python3.7/site-packages/psycopg2/extras.py", line 1299, in execute_values
    cur.execute(b''.join(parts))
psycopg2.errors.UndefinedColumn: column "id.1" of relation "algolia_tests" does not exist
LINE 1: INSERT INTO algolia_tests (index, id, "id.1", shop_domain, a...
                                              ^


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/decorators/base.py", line 188, in execute
    return_value = super().execute(context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 193, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/algolia-dag.py", line 60, in load
    df.to_sql('algolia_tests', engine, if_exists='append')
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/core/generic.py", line 2882, in to_sql
    method=method,
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/io/sql.py", line 728, in to_sql
    **engine_kwargs,
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/io/sql.py", line 1770, in to_sql
    **engine_kwargs,
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/io/sql.py", line 1350, in insert_records
    raise err
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/io/sql.py", line 1340, in insert_records
    table.insert(chunksize=chunksize, method=method)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/io/sql.py", line 967, in insert
    exec_insert(conn, keys, chunk_iter)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/io/sql.py", line 882, in _execute_insert
    conn.execute(self.table.insert(), data)
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1289, in execute
    return meth(self, multiparams, params, _EMPTY_EXECUTION_OPTS)
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/sql/elements.py", line 326, in _execute_on_connection
    self, multiparams, params, execution_options
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1491, in _execute_clauseelement
    cache_hit=cache_hit,
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1846, in _execute_context
    e, statement, parameters, cursor, context
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 2027, in _handle_dbapi_exception
    sqlalchemy_exception, with_traceback=exc_info[2], from_=e
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/util/compat.py", line 207, in raise_
    raise exception
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1783, in _execute_context
    cursor, statement, parameters, context
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/dialects/postgresql/psycopg2.py", line 957, in do_executemany
    **kwargs
  File "/home/airflow/.local/lib/python3.7/site-packages/psycopg2/extras.py", line 1299, in execute_values
    cur.execute(b''.join(parts))
sqlalchemy.exc.ProgrammingError: (psycopg2.errors.UndefinedColumn) column "id.1" of relation "algolia_tests" does not exist
LINE 1: INSERT INTO algolia_tests (index, id, "id.1", shop_domain, a...
                                              ^

[SQL: INSERT INTO algolia_tests (index, id, "id.1", shop_domain, application_id, autocomplete_enabled, user_created_at_least_one_qr, nbr_merchandised_queries, nbrs_pinned_items, showing_logo, has_changed_sort_orders, analytics_enabled, use_metafields, nbr_metafields, use_default_colors, show_products, instant_search_enabled, instant_search_enabled_on_collection, only_using_faceting_on_collection, use_merchandising_for_collection, index_prefix, indexing_paused, install_channel, export_date, has_specific_prefix) VALUES (%(index)s, %(id)s, %(id.1)s, %(shop_domain)s, %(application_id)s, %(autocomplete_enabled)s, %(user_created_at_least_one_qr)s, %(nbr_merchandised_queries)s, %(nbrs_pinned_items)s, %(showing_logo)s, %(has_changed_sort_orders)s, %(analytics_enabled)s, %(use_metafields)s, %(nbr_metafields)s, %(use_default_colors)s, %(show_products)s, %(instant_search_enabled)s, %(instant_search_enabled_on_collection)s, %(only_using_faceting_on_collection)s, %(use_merchandising_for_collection)s, %(index_prefix)s, %(indexing_paused)s, %(install_channel)s, %(export_date)s, %(has_specific_prefix)s)]
[parameters: ({'index': 0, 'id': 0, 'id.1': '1d948de7-ea94-4e00-b784-1b598c97dbaa', 'shop_domain': 'himself-hair-southern.myshopify.com', 'application_id': 'HM2DF8P5KS', 'autocomplete_enabled': True, 'user_created_at_least_one_qr': False, 'nbr_merchandised_queries': 0, 'nbrs_pinned_items': '[0]', 'showing_logo': True, 'has_changed_sort_orders': False, 'analytics_enabled': False, 'use_metafields': False, 'nbr_metafields': 9.0, 'use_default_colors': True, 'show_products': False, 'instant_search_enabled': False, 'instant_search_enabled_on_collection': False, 'only_using_faceting_on_collection': False, 'use_merchandising_for_collection': False, 'index_prefix': 'shopify_already_', 'indexing_paused': False, 'install_channel': 'marketplace', 'export_date': '2019-04-02', 'has_specific_prefix': True}, {'index': 1, 'id': 1, 'id.1': '9e1209d3-6a7d-45fa-b330-a452316f9dcc', 'shop_domain': 'professional-really-music.myshopify.com', 'application_id': 'WTE6P6DQ90', 'autocomplete_enabled': True, 'user_created_at_least_one_qr': False, 'nbr_merchandised_queries': 0, 'nbrs_pinned_items': '[0]', 'showing_logo': True, 'has_changed_sort_orders': False, 'analytics_enabled': False, 'use_metafields': False, 'nbr_metafields': 11.0, 'use_default_colors': True, 'show_products': False, 'instant_search_enabled': False, 'instant_search_enabled_on_collection': False, 'only_using_faceting_on_collection': False, 'use_merchandising_for_collection': False, 'index_prefix': 'shopify_', 'indexing_paused': False, 'install_channel': 'marketplace', 'export_date': '2019-04-02', 'has_specific_prefix': False}, {'index': 2, 'id': 2, 'id.1': 'bceefd16-ebb1-4a26-9080-38cd65b6f5ae', 'shop_domain': 'movie-than-about.myshopify.com', 'application_id': '9EMV0SZBNG', 'autocomplete_enabled': True, 'user_created_at_least_one_qr': False, 'nbr_merchandised_queries': 0, 'nbrs_pinned_items': '[0]', 'showing_logo': True, 'has_changed_sort_orders': False, 'analytics_enabled': False, 'use_metafields': False, 'nbr_metafields': 4.0, 'use_default_colors': True, 'show_products': False, 'instant_search_enabled': False, 'instant_search_enabled_on_collection': False, 'only_using_faceting_on_collection': False, 'use_merchandising_for_collection': False, 'index_prefix': 'shopify_', 'indexing_paused': False, 'install_channel': 'marketplace', 'export_date': '2019-04-02', 'has_specific_prefix': False}, {'index': 3, 'id': 3, 'id.1': '67bc3189-e1b1-4628-beb6-4dc288729c20', 'shop_domain': 'growth-this-identify.myshopify.com', 'application_id': '9TCETJH192', 'autocomplete_enabled': True, 'user_created_at_least_one_qr': False, 'nbr_merchandised_queries': 0, 'nbrs_pinned_items': '[0]', 'showing_logo': True, 'has_changed_sort_orders': False, 'analytics_enabled': False, 'use_metafields': False, 'nbr_metafields': 3.0, 'use_default_colors': True, 'show_products': False, 'instant_search_enabled': False, 'instant_search_enabled_on_collection': False, 'only_using_faceting_on_collection': False, 'use_merchandising_for_collection': False, 'index_prefix': 'shopify_his_chance_husband_', 'indexing_paused': False, 'install_channel': 'marketplace', 'export_date': '2019-04-02', 'has_specific_prefix': True}, {'index': 4, 'id': 4, 'id.1': 'fc91131e-1175-49a5-be3d-514816f365fd', 'shop_domain': 'do-forget-majority.myshopify.com', 'application_id': '83QW0XS6TQ', 'autocomplete_enabled': True, 'user_created_at_least_one_qr': False, 'nbr_merchandised_queries': 0, 'nbrs_pinned_items': '[0]', 'showing_logo': False, 'has_changed_sort_orders': False, 'analytics_enabled': False, 'use_metafields': False, 'nbr_metafields': 1.0, 'use_default_colors': True, 'show_products': False, 'instant_search_enabled': False, 'instant_search_enabled_on_collection': False, 'only_using_faceting_on_collection': False, 'use_merchandising_for_collection': False, 'index_prefix': 'shopify_behind_', 'indexing_paused': False, 'install_channel': 'marketplace', 'export_date': '2019-04-02', 'has_specific_prefix': True}, {'index': 5, 'id': 5, 'id.1': 'a7b0e244-d0b1-4153-ae8d-98e9cd0f272e', 'shop_domain': 'might-beautiful-art.myshopify.com', 'application_id': 'H9AKAWXJ6V', 'autocomplete_enabled': True, 'user_created_at_least_one_qr': False, 'nbr_merchandised_queries': 0, 'nbrs_pinned_items': '[0]', 'showing_logo': True, 'has_changed_sort_orders': False, 'analytics_enabled': False, 'use_metafields': False, 'nbr_metafields': 13.0, 'use_default_colors': True, 'show_products': False, 'instant_search_enabled': False, 'instant_search_enabled_on_collection': False, 'only_using_faceting_on_collection': False, 'use_merchandising_for_collection': False, 'index_prefix': 'shopify_arrive_land_everything_', 'indexing_paused': False, 'install_channel': 'marketplace', 'export_date': '2019-04-02', 'has_specific_prefix': True}, {'index': 6, 'id': 6, 'id.1': '03096398-15dc-4807-8fec-889d9697ec51', 'shop_domain': 'industry-age-base.myshopify.com', 'application_id': 'UQO5QXK4AV', 'autocomplete_enabled': True, 'user_created_at_least_one_qr': True, 'nbr_merchandised_queries': 3, 'nbrs_pinned_items': '[1, 2, 2]', 'showing_logo': True, 'has_changed_sort_orders': False, 'analytics_enabled': True, 'use_metafields': True, 'nbr_metafields': None, 'use_default_colors': True, 'show_products': False, 'instant_search_enabled': True, 'instant_search_enabled_on_collection': True, 'only_using_faceting_on_collection': False, 'use_merchandising_for_collection': False, 'index_prefix': 'shopify_', 'indexing_paused': False, 'install_channel': 'marketplace', 'export_date': '2019-04-02', 'has_specific_prefix': False}, {'index': 7, 'id': 7, 'id.1': '7dd67bc9-a94a-46c9-9e73-c6e4a9f7af7a', 'shop_domain': 'class-laugh-record.myshopify.com', 'application_id': 'NMFNXL64BX', 'autocomplete_enabled': True, 'user_created_at_least_one_qr': False, 'nbr_merchandised_queries': 0, 'nbrs_pinned_items': '[0]', 'showing_logo': False, 'has_changed_sort_orders': False, 'analytics_enabled': False, 'use_metafields': True, 'nbr_metafields': 18.0, 'use_default_colors': True, 'show_products': False, 'instant_search_enabled': False, 'instant_search_enabled_on_collection': False, 'only_using_faceting_on_collection': False, 'use_merchandising_for_collection': False, 'index_prefix': 'shopify_', 'indexing_paused': False, 'install_channel': 'marketplace', 'export_date': '2019-04-02', 'has_specific_prefix': False}  ... displaying 10 of 1142 total bound parameter sets ...  {'index': 1140, 'id': 1206, 'id.1': '453c5901-25ac-4251-8af5-d833982a1d65', 'shop_domain': 'group-red-range.myshopify.com', 'application_id': 'TMB3QF5RXL', 'autocomplete_enabled': True, 'user_created_at_least_one_qr': False, 'nbr_merchandised_queries': 0, 'nbrs_pinned_items': '[0]', 'showing_logo': True, 'has_changed_sort_orders': True, 'analytics_enabled': False, 'use_metafields': False, 'nbr_metafields': 14.0, 'use_default_colors': True, 'show_products': True, 'instant_search_enabled': True, 'instant_search_enabled_on_collection': False, 'only_using_faceting_on_collection': False, 'use_merchandising_for_collection': False, 'index_prefix': 'shopify_', 'indexing_paused': False, 'install_channel': 'unpublished', 'export_date': '2019-04-02', 'has_specific_prefix': False}, {'index': 1141, 'id': 1207, 'id.1': '577be9b8-7f52-4625-b479-aba19b1de36f', 'shop_domain': 'other-attention-cover.myshopify.com', 'application_id': 'RC89EL9QQD', 'autocomplete_enabled': True, 'user_created_at_least_one_qr': False, 'nbr_merchandised_queries': 0, 'nbrs_pinned_items': '[0]', 'showing_logo': False, 'has_changed_sort_orders': True, 'analytics_enabled': False, 'use_metafields': False, 'nbr_metafields': 4.0, 'use_default_colors': True, 'show_products': False, 'instant_search_enabled': False, 'instant_search_enabled_on_collection': False, 'only_using_faceting_on_collection': True, 'use_merchandising_for_collection': False, 'index_prefix': 'shopify_', 'indexing_paused': False, 'install_channel': 'unpublished', 'export_date': '2019-04-02', 'has_specific_prefix': False})]
(Background on this error at: https://sqlalche.me/e/14/f405)
[2023-05-23T22:33:40.622+0000] {taskinstance.py:1406} INFO - Marking task as FAILED. dag_id=algolia_test, task_id=load, execution_date=20190402T000000, start_date=20230523T223340, end_date=20230523T223340
[2023-05-23T22:33:40.656+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 140 for task load ((psycopg2.errors.UndefinedColumn) column "id.1" of relation "algolia_tests" does not exist
LINE 1: INSERT INTO algolia_tests (index, id, "id.1", shop_domain, a...
                                              ^

[SQL: INSERT INTO algolia_tests (index, id, "id.1", shop_domain, application_id, autocomplete_enabled, user_created_at_least_one_qr, nbr_merchandised_queries, nbrs_pinned_items, showing_logo, has_changed_sort_orders, analytics_enabled, use_metafields, nbr_metafields, use_default_colors, show_products, instant_search_enabled, instant_search_enabled_on_collection, only_using_faceting_on_collection, use_merchandising_for_collection, index_prefix, indexing_paused, install_channel, export_date, has_specific_prefix) VALUES (%(index)s, %(id)s, %(id.1)s, %(shop_domain)s, %(application_id)s, %(autocomplete_enabled)s, %(user_created_at_least_one_qr)s, %(nbr_merchandised_queries)s, %(nbrs_pinned_items)s, %(showing_logo)s, %(has_changed_sort_orders)s, %(analytics_enabled)s, %(use_metafields)s, %(nbr_metafields)s, %(use_default_colors)s, %(show_products)s, %(instant_search_enabled)s, %(instant_search_enabled_on_collection)s, %(only_using_faceting_on_collection)s, %(use_merchandising_for_collection)s, %(index_prefix)s, %(indexing_paused)s, %(install_channel)s, %(export_date)s, %(has_specific_prefix)s)]
[parameters: ({'index': 0, 'id': 0, 'id.1': '1d948de7-ea94-4e00-b784-1b598c97dbaa', 'shop_domain': 'himself-hair-southern.myshopify.com', 'application_id': 'HM2DF8P5KS', 'autocomplete_enabled': True, 'user_created_at_least_one_qr': False, 'nbr_merchandised_queries': 0, 'nbrs_pinned_items': '[0]', 'showing_logo': True, 'has_changed_sort_orders': False, 'analytics_enabled': False, 'use_metafields': False, 'nbr_metafields': 9.0, 'use_default_colors': True, 'show_products': False, 'instant_search_enabled': False, 'instant_search_enabled_on_collection': False, 'only_using_faceting_on_collection': False, 'use_merchandising_for_collection': False, 'index_prefix': 'shopify_already_', 'indexing_paused': False, 'install_channel': 'marketplace', 'export_date': '2019-04-02', 'has_specific_prefix': True}, {'index': 1, 'id': 1, 'id.1': '9e1209d3-6a7d-45fa-b330-a452316f9dcc', 'shop_domain': 'professional-really-music.myshopify.com', 'application_id': 'WTE6P6DQ90', 'autocomplete_enabled': True, 'user_created_at_least_one_qr': False, 'nbr_merchandised_queries': 0, 'nbrs_pinned_items': '[0]', 'showing_logo': True, 'has_changed_sort_orders': False, 'analytics_enabled': False, 'use_metafields': False, 'nbr_metafields': 11.0, 'use_default_colors': True, 'show_products': False, 'instant_search_enabled': False, 'instant_search_enabled_on_collection': False, 'only_using_faceting_on_collection': False, 'use_merchandising_for_collection': False, 'index_prefix': 'shopify_', 'indexing_paused': False, 'install_channel': 'marketplace', 'export_date': '2019-04-02', 'has_specific_prefix': False}, {'index': 2, 'id': 2, 'id.1': 'bceefd16-ebb1-4a26-9080-38cd65b6f5ae', 'shop_domain': 'movie-than-about.myshopify.com', 'application_id': '9EMV0SZBNG', 'autocomplete_enabled': True, 'user_created_at_least_one_qr': False, 'nbr_merchandised_queries': 0, 'nbrs_pinned_items': '[0]', 'showing_logo': True, 'has_changed_sort_orders': False, 'analytics_enabled': False, 'use_metafields': False, 'nbr_metafields': 4.0, 'use_default_colors': True, 'show_products': False, 'instant_search_enabled': False, 'instant_search_enabled_on_collection': False, 'only_using_faceting_on_collection': False, 'use_merchandising_for_collection': False, 'index_prefix': 'shopify_', 'indexing_paused': False, 'install_channel': 'marketplace', 'export_date': '2019-04-02', 'has_specific_prefix': False}, {'index': 3, 'id': 3, 'id.1': '67bc3189-e1b1-4628-beb6-4dc288729c20', 'shop_domain': 'growth-this-identify.myshopify.com', 'application_id': '9TCETJH192', 'autocomplete_enabled': True, 'user_created_at_least_one_qr': False, 'nbr_merchandised_queries': 0, 'nbrs_pinned_items': '[0]', 'showing_logo': True, 'has_changed_sort_orders': False, 'analytics_enabled': False, 'use_metafields': False, 'nbr_metafields': 3.0, 'use_default_colors': True, 'show_products': False, 'instant_search_enabled': False, 'instant_search_enabled_on_collection': False, 'only_using_faceting_on_collection': False, 'use_merchandising_for_collection': False, 'index_prefix': 'shopify_his_chance_husband_', 'indexing_paused': False, 'install_channel': 'marketplace', 'export_date': '2019-04-02', 'has_specific_prefix': True}, {'index': 4, 'id': 4, 'id.1': 'fc91131e-1175-49a5-be3d-514816f365fd', 'shop_domain': 'do-forget-majority.myshopify.com', 'application_id': '83QW0XS6TQ', 'autocomplete_enabled': True, 'user_created_at_least_one_qr': False, 'nbr_merchandised_queries': 0, 'nbrs_pinned_items': '[0]', 'showing_logo': False, 'has_changed_sort_orders': False, 'analytics_enabled': False, 'use_metafields': False, 'nbr_metafields': 1.0, 'use_default_colors': True, 'show_products': False, 'instant_search_enabled': False, 'instant_search_enabled_on_collection': False, 'only_using_faceting_on_collection': False, 'use_merchandising_for_collection': False, 'index_prefix': 'shopify_behind_', 'indexing_paused': False, 'install_channel': 'marketplace', 'export_date': '2019-04-02', 'has_specific_prefix': True}, {'index': 5, 'id': 5, 'id.1': 'a7b0e244-d0b1-4153-ae8d-98e9cd0f272e', 'shop_domain': 'might-beautiful-art.myshopify.com', 'application_id': 'H9AKAWXJ6V', 'autocomplete_enabled': True, 'user_created_at_least_one_qr': False, 'nbr_merchandised_queries': 0, 'nbrs_pinned_items': '[0]', 'showing_logo': True, 'has_changed_sort_orders': False, 'analytics_enabled': False, 'use_metafields': False, 'nbr_metafields': 13.0, 'use_default_colors': True, 'show_products': False, 'instant_search_enabled': False, 'instant_search_enabled_on_collection': False, 'only_using_faceting_on_collection': False, 'use_merchandising_for_collection': False, 'index_prefix': 'shopify_arrive_land_everything_', 'indexing_paused': False, 'install_channel': 'marketplace', 'export_date': '2019-04-02', 'has_specific_prefix': True}, {'index': 6, 'id': 6, 'id.1': '03096398-15dc-4807-8fec-889d9697ec51', 'shop_domain': 'industry-age-base.myshopify.com', 'application_id': 'UQO5QXK4AV', 'autocomplete_enabled': True, 'user_created_at_least_one_qr': True, 'nbr_merchandised_queries': 3, 'nbrs_pinned_items': '[1, 2, 2]', 'showing_logo': True, 'has_changed_sort_orders': False, 'analytics_enabled': True, 'use_metafields': True, 'nbr_metafields': None, 'use_default_colors': True, 'show_products': False, 'instant_search_enabled': True, 'instant_search_enabled_on_collection': True, 'only_using_faceting_on_collection': False, 'use_merchandising_for_collection': False, 'index_prefix': 'shopify_', 'indexing_paused': False, 'install_channel': 'marketplace', 'export_date': '2019-04-02', 'has_specific_prefix': False}, {'index': 7, 'id': 7, 'id.1': '7dd67bc9-a94a-46c9-9e73-c6e4a9f7af7a', 'shop_domain': 'class-laugh-record.myshopify.com', 'application_id': 'NMFNXL64BX', 'autocomplete_enabled': True, 'user_created_at_least_one_qr': False, 'nbr_merchandised_queries': 0, 'nbrs_pinned_items': '[0]', 'showing_logo': False, 'has_changed_sort_orders': False, 'analytics_enabled': False, 'use_metafields': True, 'nbr_metafields': 18.0, 'use_default_colors': True, 'show_products': False, 'instant_search_enabled': False, 'instant_search_enabled_on_collection': False, 'only_using_faceting_on_collection': False, 'use_merchandising_for_collection': False, 'index_prefix': 'shopify_', 'indexing_paused': False, 'install_channel': 'marketplace', 'export_date': '2019-04-02', 'has_specific_prefix': False}  ... displaying 10 of 1142 total bound parameter sets ...  {'index': 1140, 'id': 1206, 'id.1': '453c5901-25ac-4251-8af5-d833982a1d65', 'shop_domain': 'group-red-range.myshopify.com', 'application_id': 'TMB3QF5RXL', 'autocomplete_enabled': True, 'user_created_at_least_one_qr': False, 'nbr_merchandised_queries': 0, 'nbrs_pinned_items': '[0]', 'showing_logo': True, 'has_changed_sort_orders': True, 'analytics_enabled': False, 'use_metafields': False, 'nbr_metafields': 14.0, 'use_default_colors': True, 'show_products': True, 'instant_search_enabled': True, 'instant_search_enabled_on_collection': False, 'only_using_faceting_on_collection': False, 'use_merchandising_for_collection': False, 'index_prefix': 'shopify_', 'indexing_paused': False, 'install_channel': 'unpublished', 'export_date': '2019-04-02', 'has_specific_prefix': False}, {'index': 1141, 'id': 1207, 'id.1': '577be9b8-7f52-4625-b479-aba19b1de36f', 'shop_domain': 'other-attention-cover.myshopify.com', 'application_id': 'RC89EL9QQD', 'autocomplete_enabled': True, 'user_created_at_least_one_qr': False, 'nbr_merchandised_queries': 0, 'nbrs_pinned_items': '[0]', 'showing_logo': False, 'has_changed_sort_orders': True, 'analytics_enabled': False, 'use_metafields': False, 'nbr_metafields': 4.0, 'use_default_colors': True, 'show_products': False, 'instant_search_enabled': False, 'instant_search_enabled_on_collection': False, 'only_using_faceting_on_collection': True, 'use_merchandising_for_collection': False, 'index_prefix': 'shopify_', 'indexing_paused': False, 'install_channel': 'unpublished', 'export_date': '2019-04-02', 'has_specific_prefix': False})]
(Background on this error at: https://sqlalche.me/e/14/f405); 1722)
[2023-05-23T22:33:40.691+0000] {local_task_job.py:159} INFO - Task exited with return code 1
[2023-05-23T22:33:40.712+0000] {taskinstance.py:2623} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-05-23T22:38:45.387+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: algolia_test.load scheduled__2019-04-02T00:00:00+00:00 [queued]>
[2023-05-23T22:38:45.409+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: algolia_test.load scheduled__2019-04-02T00:00:00+00:00 [queued]>
[2023-05-23T22:38:45.413+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-05-23T22:38:45.414+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-05-23T22:38:45.419+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-05-23T22:38:45.467+0000] {taskinstance.py:1383} INFO - Executing <Task(_PythonDecoratedOperator): load> on 2019-04-02 00:00:00+00:00
[2023-05-23T22:38:45.496+0000] {standard_task_runner.py:55} INFO - Started process 2043 to run task
[2023-05-23T22:38:45.514+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'algolia_test', 'load', 'scheduled__2019-04-02T00:00:00+00:00', '--job-id', '160', '--raw', '--subdir', 'DAGS_FOLDER/algolia-dag.py', '--cfg-path', '/tmp/tmpuxz9f7jf']
[2023-05-23T22:38:45.525+0000] {standard_task_runner.py:83} INFO - Job 160: Subtask load
[2023-05-23T22:38:45.553+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:545: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  option = self._get_environment_variables(deprecated_key, deprecated_section, key, section)

[2023-05-23T22:38:45.672+0000] {task_command.py:376} INFO - Running <TaskInstance: algolia_test.load scheduled__2019-04-02T00:00:00+00:00 [running]> on host 7bd2786c5ce5
[2023-05-23T22:38:45.762+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=algolia_test
AIRFLOW_CTX_TASK_ID=load
AIRFLOW_CTX_EXECUTION_DATE=2019-04-02T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-04-02T00:00:00+00:00
[2023-05-23T22:38:45.763+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:204: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2023-05-23T22:38:45.778+0000] {logging_mixin.py:137} INFO - Index(['index', 'id', 'shop_domain', 'application_id', 'autocomplete_enabled',
       'user_created_at_least_one_qr', 'nbr_merchandised_queries',
       'nbrs_pinned_items', 'showing_logo', 'has_changed_sort_orders',
       'analytics_enabled', 'use_metafields', 'nbr_metafields',
       'use_default_colors', 'show_products', 'instant_search_enabled',
       'instant_search_enabled_on_collection',
       'only_using_faceting_on_collection', 'use_merchandising_for_collection',
       'index_prefix', 'indexing_paused', 'install_channel', 'export_date',
       'has_specific_prefix'],
      dtype='object')
[2023-05-23T22:38:45.890+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-05-23T22:38:45.900+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=algolia_test, task_id=load, execution_date=20190402T000000, start_date=20230523T223845, end_date=20230523T223845
[2023-05-23T22:38:45.956+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2023-05-23T22:38:45.973+0000] {taskinstance.py:2623} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-05-24T10:56:21.831+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: algolia_test.load scheduled__2019-04-02T00:00:00+00:00 [queued]>
[2023-05-24T10:56:21.846+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: algolia_test.load scheduled__2019-04-02T00:00:00+00:00 [queued]>
[2023-05-24T10:56:21.847+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-05-24T10:56:21.847+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-05-24T10:56:21.848+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-05-24T10:56:21.867+0000] {taskinstance.py:1383} INFO - Executing <Task(_PythonDecoratedOperator): load> on 2019-04-02 00:00:00+00:00
[2023-05-24T10:56:21.874+0000] {standard_task_runner.py:55} INFO - Started process 5107 to run task
[2023-05-24T10:56:21.886+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'algolia_test', 'load', 'scheduled__2019-04-02T00:00:00+00:00', '--job-id', '177', '--raw', '--subdir', 'DAGS_FOLDER/algolia-dag.py', '--cfg-path', '/tmp/tmpxkp12qcp']
[2023-05-24T10:56:21.900+0000] {standard_task_runner.py:83} INFO - Job 177: Subtask load
[2023-05-24T10:56:21.929+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:545: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  option = self._get_environment_variables(deprecated_key, deprecated_section, key, section)

[2023-05-24T10:56:22.037+0000] {task_command.py:376} INFO - Running <TaskInstance: algolia_test.load scheduled__2019-04-02T00:00:00+00:00 [running]> on host 7bd2786c5ce5
[2023-05-24T10:56:22.136+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=algolia_test
AIRFLOW_CTX_TASK_ID=load
AIRFLOW_CTX_EXECUTION_DATE=2019-04-02T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-04-02T00:00:00+00:00
[2023-05-24T10:56:22.138+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:204: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2023-05-24T10:56:22.352+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-05-24T10:56:22.371+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=algolia_test, task_id=load, execution_date=20190402T000000, start_date=20230524T105621, end_date=20230524T105622
[2023-05-24T10:56:22.463+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2023-05-24T10:56:22.517+0000] {taskinstance.py:2623} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-05-24T11:29:41.213+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: algolia_test.load scheduled__2019-04-02T00:00:00+00:00 [queued]>
[2023-05-24T11:29:41.222+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: algolia_test.load scheduled__2019-04-02T00:00:00+00:00 [queued]>
[2023-05-24T11:29:41.222+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-05-24T11:29:41.223+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-05-24T11:29:41.223+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-05-24T11:29:41.236+0000] {taskinstance.py:1383} INFO - Executing <Task(_PythonDecoratedOperator): load> on 2019-04-02 00:00:00+00:00
[2023-05-24T11:29:41.243+0000] {standard_task_runner.py:55} INFO - Started process 6942 to run task
[2023-05-24T11:29:41.254+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'algolia_test', 'load', 'scheduled__2019-04-02T00:00:00+00:00', '--job-id', '224', '--raw', '--subdir', 'DAGS_FOLDER/algolia-dag.py', '--cfg-path', '/tmp/tmpdl7dtkvo']
[2023-05-24T11:29:41.263+0000] {standard_task_runner.py:83} INFO - Job 224: Subtask load
[2023-05-24T11:29:41.352+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:545: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  option = self._get_environment_variables(deprecated_key, deprecated_section, key, section)

[2023-05-24T11:29:41.512+0000] {task_command.py:376} INFO - Running <TaskInstance: algolia_test.load scheduled__2019-04-02T00:00:00+00:00 [running]> on host 7bd2786c5ce5
[2023-05-24T11:29:41.592+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=algolia_test
AIRFLOW_CTX_TASK_ID=load
AIRFLOW_CTX_EXECUTION_DATE=2019-04-02T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-04-02T00:00:00+00:00
[2023-05-24T11:29:41.594+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:204: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2023-05-24T11:29:41.710+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-05-24T11:29:41.727+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=algolia_test, task_id=load, execution_date=20190402T000000, start_date=20230524T112941, end_date=20230524T112941
[2023-05-24T11:29:41.759+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2023-05-24T11:29:41.782+0000] {taskinstance.py:2623} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-05-26T12:12:33.031+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: algolia_test.load scheduled__2019-04-02T00:00:00+00:00 [queued]>
[2023-05-26T12:12:33.058+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: algolia_test.load scheduled__2019-04-02T00:00:00+00:00 [queued]>
[2023-05-26T12:12:33.060+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-05-26T12:12:33.062+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-05-26T12:12:33.064+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-05-26T12:12:33.089+0000] {taskinstance.py:1383} INFO - Executing <Task(_PythonDecoratedOperator): load> on 2019-04-02 00:00:00+00:00
[2023-05-26T12:12:33.096+0000] {standard_task_runner.py:55} INFO - Started process 567 to run task
[2023-05-26T12:12:33.115+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'algolia_test', 'load', 'scheduled__2019-04-02T00:00:00+00:00', '--job-id', '245', '--raw', '--subdir', 'DAGS_FOLDER/algolia-dag.py', '--cfg-path', '/tmp/tmp_1n3q48x']
[2023-05-26T12:12:33.144+0000] {standard_task_runner.py:83} INFO - Job 245: Subtask load
[2023-05-26T12:12:33.217+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:545: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  option = self._get_environment_variables(deprecated_key, deprecated_section, key, section)

[2023-05-26T12:12:33.337+0000] {task_command.py:376} INFO - Running <TaskInstance: algolia_test.load scheduled__2019-04-02T00:00:00+00:00 [running]> on host b55299475a02
[2023-05-26T12:12:33.431+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=algolia_test
AIRFLOW_CTX_TASK_ID=load
AIRFLOW_CTX_EXECUTION_DATE=2019-04-02T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-04-02T00:00:00+00:00
[2023-05-26T12:12:33.496+0000] {taskinstance.py:1851} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/decorators/base.py", line 188, in execute
    return_value = super().execute(context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 193, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/algolia-dag.py", line 51, in load
    postgres_conn = BaseHook.get_connection("postgres_conn")
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/hooks/base.py", line 70, in get_connection
    conn = Connection.get_connection_from_secrets(conn_id)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/connection.py", line 432, in get_connection_from_secrets
    raise AirflowNotFoundException(f"The conn_id `{conn_id}` isn't defined")
airflow.exceptions.AirflowNotFoundException: The conn_id `postgres_conn` isn't defined
[2023-05-26T12:12:33.504+0000] {taskinstance.py:1406} INFO - Marking task as FAILED. dag_id=algolia_test, task_id=load, execution_date=20190402T000000, start_date=20230526T121233, end_date=20230526T121233
[2023-05-26T12:12:33.520+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 245 for task load (The conn_id `postgres_conn` isn't defined; 567)
[2023-05-26T12:12:33.571+0000] {local_task_job.py:159} INFO - Task exited with return code 1
[2023-05-26T12:12:33.606+0000] {taskinstance.py:2623} INFO - 0 downstream tasks scheduled from follow-on schedule check
